src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String machineType) {
+        super(machineType);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer" + " " + machineType + " " + "started its work.");

```

Comment:
The message in the `doWork` method should be more informative and indicate that the Bulldozer is working. Consider including the specific action it performs, for example, 'Bulldozer is moving earth'.